### ISSUE TITLE
Succesfully adding a new contact to an existing list causes an exception

### ISSUE THREAD

**Issue by realtebo** [sendinblue/APIv3-php-library#139]
I'm using 

    "sendinblue/api-v3-sdk": "7.x.x",

resolved as v7.3.0

The following problem happens both on PHP 5.6 and on PHP 7.2  - 7.4

I have a list, with id 5
I send a new contact, and add it to list 5

I am doing this

```
    public function addContact($email, $attributes = null, $lists = [])
    {
        $contact = new CreateContact([
            'email' => $email,
            'attributes' => $attributes
        ]);

        $this->instance->createContact($contact);

        /**
         * se passo il paarametro della lista sposto il contatto
         * nella lista indicata
         */
        if(count($lists))
        {
            $contactEmails = new AddContactToList(['emails' => [ $email ]]);
            foreach ($lists as $list)
            {
                $this->instance->addContactToList($list, $contactEmails);
            }
        }

        return true;
    }
```

where `$this->instance`  is

      $this->instance = new ContactsApi(
            new Client(),
            $config
        );

I got this exectption stack trace

```

{
    "message": "Array to string conversion",
    "exception": "ErrorException",
    "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/ObjectSerializer.php",
    "line": 271,
    "trace": [
        {
            "function": "handleError",
            "class": "Illuminate\\Foundation\\Bootstrap\\HandleExceptions",
            "type": "->"
        },
        {
            "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/ObjectSerializer.php",
            "line": 271,
            "function": "settype"
        },
        {
            "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/ObjectSerializer.php",
            "line": 252,
            "function": "deserialize",
            "class": "SendinBlue\\Client\\ObjectSerializer",
            "type": "::"
        },
        {
            "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/ObjectSerializer.php",
            "line": 316,
            "function": "deserialize",
            "class": "SendinBlue\\Client\\ObjectSerializer",
            "type": "::"
        },
        {
            "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/ObjectSerializer.php",
            "line": 316,
            "function": "deserialize",
            "class": "SendinBlue\\Client\\ObjectSerializer",
            "type": "::"
        },
        {
            "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/Api/ContactsApi.php",
            "line": 164,
            "function": "deserialize",
            "class": "SendinBlue\\Client\\ObjectSerializer",
            "type": "::"
        },
        {
            "file": "<project>/vendor/sendinblue/api-v3-sdk/lib/Api/ContactsApi.php",
            "line": 104,
            "function": "addContactToListWithHttpInfo",
            "class": "SendinBlue\\Client\\Api\\ContactsApi",
            "type": "->"
        },
        
```

I repeat:
- contact is created well
- contact is added to the right list
- in your serializer, something in the response looks like the source of this problem.

Actually I'm not still able to avoid the error and have a ML susbscription system down.

Exception is fired when `lib\Api\ContactsApi.php` reaches linea 163

            return [
                ObjectSerializer::deserialize($content, $returnType, []),
                $response->getStatusCode(),
                $response->getHeaders()
            ];

We traced the problem probably.

Calling ```https://api.sendinblue.com/v3/contacts/lists/5/contacts/add```

We got something like

```
{
    "contacts": {
        "success": {
            "ids": [],
            "emails": [
                "emailaddres@yahoo.it"
            ]
        },
        "failure": []
    }
}


```

Please note that 

```
"ids": [],
```

This causes a problem into `lib\ObjectSerializer.php` at this point

https://github.com/sendinblue/APIv3-php-library/blob/9eca20295edca98bca842dadcb2218157009c012/lib/ObjectSerializer.php#L248-L254

The code calls `self::deserialize($value, $subClass, null);` using a literal string 'Array' as `$value`

**Comment by realtebo** [comment]
TLDR: PLEASE NOTE THAT:

your documentation told about this return response

```
{
"success":[
0:"jeff32@example.com"
1:"jim56@example.com"
]
"failure":[
0:"david@example.com"
]
}
```

but this is NOT what the api returns


Your PHP SDK parses data as documented. But you're send wrong data in api responses.

Please fix this ASAP

**Comment by mikedamoiseau** [comment]
Same problem here since yesterday or the day before yesterday, not sure...

A fix would be great :)

**Comment by rajatsib** [comment]
Hi @realtebo @mikedamoiseau 

The response returned in case of emails is like below 
```
{
   "success":[
     "jeff32@example.com",
     "jim56@example.com"
   ],
   "failure":[
      "david@example.com"
    ]
}
```

and in case of ids, it's 

```
{
   "success":[
      1,
      2
    ],
    "failure":[
       5
     ]
}
```

The ids field is not returned separately in response .

**Comment by realtebo** [comment]
This was not the problem.

YOUR api, YOUR sdk, was exploding because of some not expected return from YOUR servers

I have no problem in my code.

**Comment by rajatsib** [comment]
Hi @realtebo 

This issue could not be reproduced on our side.
Can you please raise a support ticket to Sendinblue if you are still receiving the same issue
